src/Util/Crypto.php

```diff
@@ -146,8 +146,8 @@ public static function meToPem($m, $e)
 
 	/**
 	 * @param string $key key
-	 * @param string $m   modulo reference
-	 * @param object $e   exponent reference
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.